src/modules/transaction/components/TransactionPriority/FeeViewer.js

```diff
@@ -11,7 +11,7 @@ import { validateAmount } from 'src/utils/validators';
 import styles from './TransactionPriority.css';
 
 const getCustomFeeStatus = ({ customFeeInput, minFee, minRequiredBalance, token }) => {
-  if (!customFeeInput || !token || !minRequiredBalance) return undefined;
```

Comment:
If we remove `minRequiredBalance` check it might fail with `accountBalance: BigInt(token.availableBalance) - BigInt(minRequiredBalance),`?

Reply:
nope it does not fail